contracts/context-config/tests/sandbox.rs

```diff
@@ -37,6 +41,17 @@ async fn main() -> eyre::Result<()> {
         .await?
         .into_result()?;
 
+    // Fund both nodes with enough NEAR
+    let _tx1 = root_account
+        .transfer_near(node1.id(), NearToken::from_near(1))
+        .await?
+        .into_result()?;
+
+    let _tx2 = root_account
+        .transfer_near(node2.id(), NearToken::from_near(1))
+        .await?
+        .into_result()?;
```

Comment:
I think above you can make it `root_account.create_subaccount(..).initial_balance()` or something like that if i remember correctly